.github/RELEASE_CHECKLIST.md

```diff
@@ -37,5 +37,10 @@
 2. Close milestone in issues.
 3. Publish release on GitHub.
 4. Announce it on Slack.
-5. Bump the latest version on [Beacon](https://beacon.ferretdb.io).
-6. Publish blog post, etc.
+
+## Soon after
+
+1. Bump the latest version on [Beacon](https://beacon.ferretdb.io).
+2. Publish and announce blog post.
+3. Tweet, toot.
+4. Update NixOS package.
```

Comment:
That would be great also to have documentation things like "Bump the latest version on Beacon" and "Update NixOS package", but as far as I understand, it's a checklist at the moment.

Reply:
It is also more-or-less a personal checklist — most things could be done only by me, but that’s changing slowly